packages/flutter/lib/src/gestures/force_press.dart

```diff
@@ -0,0 +1,262 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show Offset;
+
+import 'package:flutter/foundation.dart';
+
+import 'arena.dart';
+import 'constants.dart';
+import 'events.dart';
+import 'recognizer.dart';
+
+enum _ForceState {
+  // No pointer has touched down and the detector is ready for a pointer down to occur.
+  ready,
+
+  // A pointer has touched down, but a force press gesture has not yet been detected.
+  possible,
+
+  // A pointer is down and a force press gesture has been detected. However, if
+  // ForcePressGestureRecognizer the only recognizer in the arena, the gesture
```

Comment:
Not sure I understand this comment. What makes the gesture get accepted if it hasn't started yet?

Reply:
So apparently gestures will be accepted by default if they're the only gestures in the arena.

Reply:
I would write:
```
However, if the ForcePressGestureRecognizer is the only recognizer in the arena, thus accepted as soon as the gesture state is possible, the gesture will not yet have started.
```
to connect the relationship